test/e2e/specs/mainPageVideoIsPlaying.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { vpTest } from '../fixtures/vpTest';
+import { expect, test } from '@playwright/test';
+import { waitForPageToLoadWithTimeout } from '../src/helpers/waitForPageToLoadWithTimeout';
+
+/**
+ * Testing if video on main page is playing by checking that is pause return false.
+ * The video in the main page is not configured as autoplay so first need to click on play button.
+ */
+vpTest(`Test if video on main page is playing`, async ({ page, vpExamples }) => {
```

Comment:
`Test if video on main page can play as expected` since it is not an autoplay i think such a msg is more clear